bears/rest/RSTcheckBear.py

```diff
@@ -0,0 +1,58 @@
+
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class RSTcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck', '2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file,
+            code_block_language_ignore: list):
+        """
+        A Bear for rstcheck. Check
+        <https://pypi.python.org/pypi/rstcheck> for more information on
+        rstcheck.
+        Rstcheck checks syntax of reStructuredText and code blocks
+        nested within it.
+        Supports code blocks of ``bash``, ``c``, ``cpp``, ``json``,
+        ``python`` and ``rst``.
+
+        :param code_block_language_ignore:
+            Comma seperated list for languages of which code blocks
+            should be ignored. Code block of following languages
+            can be detected: ``bash``, ``c``, ``cpp``, ``json``,
+            ``python``, ``rst``.
+        """
+        for error in check(''.join(file), filename=filename,
+                           ignore=code_block_language_ignore):
+            """
+            rstcheck has 4 levels of reports info, warning, normal and
+            severe, extracting level from error message to map it to
+            coala severity level
+            """
+            try:
+                report_level = int(
+                    re.search(r'\(\w+\/([0-9])\)', error[1]).group(1))
+            except AttributeError:
+                report_level = 3  # Default is 3
+
+            report_level = 0 if report_level < 3 else report_level-2
```

Comment:
why are you using hardcoded numbers here? The numbers may change, please use `RESULT_SEVERITY....` instead to make your code "bulletproof"